config.json

```diff
@@ -1,6 +1,6 @@
 {
     "UseFoodPerAgentRatio": false,
-    "FoodOnPlatform": 100,
+    "FoodOnPlatform": 10,
```

Comment:
Prefer to leave config as is!

Reply:
Yes that's on me - I thought I updated from main but perhaps not!